common/fw-update-helper.h

```diff
@@ -10,12 +10,16 @@ namespace rs2
 {
     class viewer_model;
 
-    int parse_product_line(std::string id);
-    std::string get_available_firmware_version(int product_line, std::string PID);
+    int parse_product_line(const std::string& id);
+    std::string get_available_firmware_version(int product_line, const std::string& PID);
+
+    // return map from product line+PID to default fw image
```

Comment:
product line+PID to FW image data
Remove 3rd line

Reply:
done